client/src/components/Dashboard/AddItem.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState, useEffect } from 'react';
+import {
+  makeStyles,
+  MuiThemeProvider,
+  createMuiTheme,
+} from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import Input from '@material-ui/core/Input';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import Typography from '@material-ui/core/Typography';
+
+const addItemTheme = createMuiTheme({
+  palette: {
+    primary: { main: '#DF1B1B' },
+  },
+});
+
+const mockData = [
+  { title: 'Scented Candles' },
+  { title: 'Towels' },
+  { title: 'Kale' },
+];
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+  },
+  label: {
+    margin: 50,
+  },
+  input: {
+    width: 400,
+    height: 30,
+    maxWidth: '100%',
+    backgroundColor: 'white',
+    fontFamily: theme.typography.fontFamily,
+    fontSize: '1.6rem',
+    padding: '15px 25px',
+    borderRadius: '28px 0 0 28px',
+    boxShadow: theme.boxShadowTheme,
+  },
+  selectContainer: {
+    display: 'flex',
+    justifyContent: 'space-around',
+    width: 320,
+    maxWidth: '100%',
+    backgroundColor: 'white',
+    padding: 5,
+    borderRadius: '0 28px 28px 0',
+    boxShadow: theme.boxShadowTheme,
+  },
+  select: {
+    width: 180,
+    maxWidth: '100%',
+    padding: 10,
+    fontFamily: theme.typography.fontFamily,
+    fontSize: '1.6rem',
+  },
+  addButton: {
+    fontSize: '1.6rem',
+    borderRadius: 24,
+    width: 140,
+    position: 'relative',
+    left: '140',
+  },
+  menuItem: {
+    fontSize: '1.6rem',
+  },
+}));
+
+export default function AddItem() {
+  const classes = useStyles();
+  const [input, set] = useState({ text: '', select: '' });
+  const [options, setOptions] = useState([]);
+
+  const handleInputChange = e => {
+    set({ text: e.target.value });
+  };
+
+  const handleSelectChange = e => {
+    set({ select: e.target.value });
+  };
```

Comment:
Updating state via useState and setState are different when it comes to objects
With setState:

- Passing an object argument will update only the properties in said object

With useState:

- The object argument replaces the previous state entirely

So the eventHandlers on lines 79 and 83 will update text and select properties respectively, but throw out anything else not in the object argument. This is not what we want. A better way might be to use useState() to keep track of a single value instead of objects. Here's a [link](https://blog.logrocket.com/a-guide-to-usestate-in-react-ecb9952e406c/) that might clear some confusion regarding useState.

The naming convention with useState is
``
const [varname, setVarname] = useState('')
``

The eventHandlers can be passed as anonymous functions since they are simple.